javascript-methods/main.js

```diff
@@ -0,0 +1,53 @@
+var a = 7;
+var b = 8;
+var c = 10;
+var maximumValue = Math.max(a, b, c);
+console.log('value of maximumValue:', maximumValue);
+var heroes = ['Thor', 'Spiderman', 'Batman', 'Black Panther'];
+var randomNumber = Math.random();
+randomNumber *= heroes.length;
+var randomIndex = Math.floor(randomNumber);
+console.log('value of randomIndex:', randomIndex);
+var randomHero = heroes[randomIndex];
+console.log('value of randomHero:', randomHero);
+
+var library = [
+  {
+    title: 'Harry Potter',
+    author: 'J.K. Rowling'
+  },
+  {
+    title: "Can't hurt me",
+    author: 'David Goggins'
+  },
+  {
+    title: 'Green Eggs and Ham',
+    author: 'Dr. Seuss'
+  }
+];
+
+var lastBook = library.pop();
+console.log('value of lastBook:', lastBook);
+var firstBook = library.shift();
+console.log('value of firstBook:', firstBook);
+
+var js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer'
+};
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+library.unshift(css);
+library.splice(1);
```

Comment:
You're missing a 2nd argument here. Right now you're deleting everything from the array after the 1st index. Check your console log and compare it to the example output

Reply:
<img width="752" alt="Screenshot 2023-02-01 at 10 05 49 AM" src="https://user-images.githubusercontent.com/122064415/216126532-460d80e0-41e2-464c-9ce7-4106695fb8f0.png">